quarkchain/cluster/master.py

```diff
@@ -759,6 +771,9 @@ def __init__(self, env, root_state, name="master"):
         # branch value -> a list of slave running the shard
         self.branch_to_slaves = dict()  # type: Dict[int, List[SlaveConnection]]
         self.slave_pool = set()
+        self.grpc_server = (
```

Comment:
do we need a store grpc server into master?

Reply:
I try to store it in elsewhere, but, it doesn't work, since we have to keep grpc server alive

Reply:
already move out